Translation corrections (#451)
07th-mod/higurashi-console-arcs: Update/ura_014.txt

//r…ということは、その取引現場を押さえることができれば、まさに起死回生の大逆転劇となるだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…ということは、その取引現場を押さえることができれば、まさに起死回生の大逆転劇となるだろう。",

> 			NULL, "...Which meant that if they find out where the trade would be held, they'd be delivering a miraculous come-from-behind counterattack.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Which meant that if they can identify the trade location, they'd be delivering a miraculous come-from-behind counterattack.", Line_Normal);
```
